packages/palette/src/elements/Grid/Grid.tsx

```diff
@@ -1,22 +1,14 @@
-import { color, flex, maxWidth, space, textAlign, width } from "styled-system"
-
 import {
   Col as _Col,
   Container as _Container,
   Row as _Row,
 } from "styled-bootstrap-grid"
-
 import styled from "styled-components"
+import { color, flex, maxWidth, space, textAlign, width } from "styled-system"
 
-/**
- * TODO: v2 of `styled-bootstrap-grid` contains TS typings, but we need to
- * upgrade to styled-components 4 before it's possible to upgrade to v2.
- */
-
-/** Outter wrapper when using a grid */
+/** Outer wrapper when using a grid */
 export const Grid: any = styled(_Container)`
-  overflow: hidden;
-  max-width: ${props => props.theme.grid.breakpoints.xl};
+  max-width: ${props => props.theme.grid.breakpoints.xl}px;
```

Comment:
Minor bug here. Apparently this doesn't matter though, we never use this as an outer container on it's own.